README.md

```diff
@@ -412,6 +413,23 @@ list.map({ item => ... })
 
 ## <a name='lang'>Scala Language Features</a>
 
+### <a name='case_class_immutability'>Case Classes and Immutability</a>
+
+Case classes are regular classes but extended by the compiler to automatically support:
+- Public getters for constructor parameters
+- Copy constructor
+- Pattern matching on constructor parameters
+- Automatic toString/hash/equals implementation
+
+Constructor parameters should NOT be mutable for case classes.
+```scala
+// This is OK
+case class Person(name: String, age: Int)
+
+// This is NOT OK
```

Comment:
I actually have do this.  I think its only a bad idea if the case class is not local (and in general I think sharing mutable data structures across boundaries is a bad idea)